plano

```diff
@@ -24,6 +24,8 @@ import sys
 if os.path.isdir("python"):
     sys.path.insert(0, "python")
 
+from plano import PlanoCommand
```

Comment:
This is essentially revert of https://github.com/ssorj/blinky/commit/7d29bee2d31906188f9f5d1372b0e06502ffd546, so probably there is something I'm missing here